clusterDeployer.py

```diff
@@ -135,6 +139,25 @@ def setup_all_vms(h: host.Host, vms: List[NodeConfig], iso_path: str) -> List[Fu
 
     return futures
 
+def deploy_microshift(cluster_name:str, nodes: List[NodeConfig] ,external_port:str) -> None:
+    lh = host.LocalHost()
+    bmc = host.bmc_from_host_name_or_ip(nodes[0].node, nodes[0].bmc_ip, nodes[0].bmc_user, nodes[0].bmc_password)
+    h = Host(nodes[0].node, bmc)
+    try:
+        microshift_iso_builder(lh)
+    except Exception as e:
+        logger.error(f"ISO was not able to be created")
+
+    if os.path.exists("./final.iso"):
+        logger.info(f"The file 'final.iso' exists.")
+    else:
+        logger.info(f"The file 'final.iso' does not exist.")
```

Comment:
need to bail out in case it doesn't exist since the code below fille fail. Look at how we bail out in other places.